modules/rosa-hcp-sharing-vpc-hosted-zones.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-hosted-zones_{context}"]
+= Step Three - VPC Owner: Updating the shared VPC role and creating hosted zones
+
+After the *Cluster Creator* provides the DNS domain and the IAM roles, create two hosted zones and update the trust policy on the IAM roles that were created for sharing the VPC.
+
+[NOTE]
+====
+The hosted zones can be created in either the centrally-managed VPC account or in the workload account.
+====
+
+image::522-shared-vpc-step-3.png[]
+
+* The hosted zones can be created in either the centrally-managed VPC account or in the workload account in which the cluster is deployed.
+
+.Prerequisites
+
+* You have the full domain name from the *Cluster Creator*.
+* You have the _Ingress Operator Cloud Credentials_ role's ARN from the *Cluster Creator*.
+* You have the _Installer_ role's ARN from the *Cluster Creator*.
+* You have the _Control plane Operator Cloud Credentials_ role's ARN from the *Cluster Creator*.
+
+.Procedure
+
+. In the link:https://console.aws.amazon.com/ram/[Resource Access Manager of the AWS console], create a resource share that shares the previously created `Route 53 Role` trust policy public and private subnets with the *Cluster Creator's* AWS account ID.
+
+. Update the VPC sharing IAM role and add the _Installer_ and _Ingress Operator Cloud Credentials_ roles to the principal section of the trust policy.
++
+[source,terminal]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+	  "Sid": "Statement1",
+	  "Effect": "Allow",
+	  "Principal": {
+	  	"AWS": [
+          "arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/<prefix>-ingress-operator-cloud-credentials",
+          "arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/<prefix>-hcp-Installer-Role"
+          "arn:aws:iam:<Cluster-Creator's-AWS-Account-ID>:role/<prefix>-control-plane-operator-cloud-credentials"
```

Comment:
Invalid, should be:
`"arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/<prefix>-control-plane-operator-cloud-credentials"`